modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc

```diff
@@ -178,21 +189,24 @@ include::assets-packages-gloss.adoc
 include::assets-creating-proc.adoc
 ----
 
-Then for cross-references to the reused modules, you specify which context (assembly) you want to link to. For example, you can link to the "Projects" module as it appears either in the "Chapter 3: Assets" assembly or in the "Chapter 4: Projects" assembly:
+For all cross-references to the reused modules, specify which context (assembly) you want to link to. For example, you can link to the "Projects" module as it appears either in the "Chapter 3: Assets" assembly or in the "Chapter 4: Projects" assembly:
 
 [source]
 ----
 For details, see xref:projects_con_assembly-assets[].
+----
 
 or
 
+[source]
+----
 For details, see xref:projects_con_assembly-projects[].
-
 ----
 
+[discrete]
```

Comment:
While I can see why we would want the practical examples numbered and shown in the ToC, I believe being able to go to that example without necessarily having read (scrolled) through the intro of the module is undesirable. However, feel free to disregard this suggestion.